index.html

```diff
@@ -889,6 +884,76 @@ <h2 class='informative'>Vary</h2>
+        rejection specified here.
+      </p>
+      <section id='message-external-body-accept-post'>
+        <p>
+          LDP-S that support LDP-NR with <code>message/external-body</code>
+          MUST advertise that support in the <code>Accept-Post</code> response
+          header for each supported <code>type</code> parameter of supported Content-Types.
+        </p>
+      </section>
+      <section id='message-external-body-unsupported-type'>
+        <p>
+          LDP-S receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> with an unsupported
+          <code>type</code> parameter MUST respond with HTTP 415 UNSUPPORTED MEDIA TYPE.
+          In the case that a Fedora LDP-S does not support external LDP-NR content, all
+          <code>message/external-body</code> messages must be rejected with HTTP 415.
+        </p>
+      </section>
+      <section id='message-external-body-response-headers'>
+        <p>
+          LDP-S receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> MUST NOT accept the
+          request if it cannot guarantee all of the response headers required
+          by the LDP-NR interaction model in this specification.
+        </p>
+      </section>
+      <section id='external-content-content-location'>
+        <p>
+          LDP-NR GET and HEAD responses SHOULD include a Content-Location: header with a URI
+          representation of the location of the external content if the LDP-S is proxying
+          the content.
+        </p>
+      </section>
+      <section id='external-content-expires'>
+        <p>
+          Per [[!RFC2017]], all <code>message/external-body</code> Content-Type values
+          may include an <code>expires</code> parameter. LDP-S receiving requests that
+          would create or update a LDP-NR with a <code>message/external-body</code> content
+          type SHOULD respect the <code>expires</code> parameter, if present, by copying
+          content. If the expires parameter cannot be accomodated, the request MUST be rejected
+          with a 4xx or 5xx status code.
+        </p>
+      </section>
+      <section id='external-content-caveats' class='informative'>
+        <h4>Referenced RDF Content in Mandatory LDP Serializations</h4>
+        <p>
+          This specification takes no position on the use of <code>message/external-body</code>
+          to create or update LDP-RS with ttl or json-ld.
+        </p>
+        <h4>Proxied Content vs Redirected Content</h4>
+        <p>
+          This specification assumes that clients interested in resolving a <code>type='url'</code>
+          or other value without the LDP-S as an intermediary (effectively, redirection as opposed
+          to proxying) will be able to negotiate the <code>Content-Location</code> response header
+          value from a HEAD request.
```

Comment:
I read this paragraph as specifying a way for a client (at its option) to use `HEAD` to decide whether it can go fetch remote content from the `Content-Location` itself, avoid a default proxy action on `GET`. It seems that from https://tools.ietf.org/html/rfc7231#section-3.1.4.2 the response to `GET` should be either:
  * 200, `Content-Location` and content in body to proxy
  * 3xx, `Content-Location` and `Location` to same URI to redirect (no body)